src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -124,6 +127,12 @@ private ExportFormat format(Object format) {
         return format != null && format instanceof String ? ExportFormat.fromString((String)format) : ExportFormat.NEO4J_SHELL;
     }
 
+    private static String[] convertCaption(Object value) {
+        if (value == null) return null;
+        List<String> strings = (List<String>) value;
```

Comment:
please check before if this is a list otherwise we could treat it as a single string or fail with an error